.circleci/config.yml

```diff
@@ -129,10 +129,11 @@ jobs:
 
   rebase-and-unit-test:
     docker:
-      - image: boltdev/m2-installed-plugin-ci-php72:2.3.0-v2
+      - image: boltdev/m2-installed-plugin-ci-php72:2.3.0-v1
```

Comment:
whats the difference between v2 and V1 here?
Why downgrade?

Reply:
This is required, otherwise memory limit breaks for some reason